concrete/controllers/dialog/type/update_from_type.php

```diff
@@ -0,0 +1,284 @@
+<?php
+
+namespace Concrete\Controller\Dialog\Type;
+
+use Concrete\Controller\Backend\UserInterface as BackendInterfaceController;
+use Concrete\Core\Page\PageList;
+use Concrete\Core\Http\ResponseFactory;
+use Concrete\Core\Foundation\Queue\Queue;
+use PageTemplate;
+use PageType;
+use Permissions;
+
+class UpdateFromType extends BackendInterfaceController
+{
+    protected $viewPath = '/dialogs/type/update_from_type';
+
+
+    private function fetchTypeAndTemplate($ptID, $pTemplateID){
+      $cmp = new \Permissions($this->pagetype);
```

Comment:
I'd say, as a best practice, to put your class dependencies (imports) on top. Btw, I'm not the one that will merge / approve your PR. Just giving you feedback on how to improve your PR.